stmt.go

```diff
@@ -65,12 +65,19 @@ func (s *Stmt) Exec(args []driver.Value) (driver.Result, error) {
 	if err != nil {
 		return nil, err
 	}
-	var c api.SQLLEN
-	ret := api.SQLRowCount(s.os.h, &c)
-	if IsError(ret) {
-		return nil, NewError("SQLRowCount", s.os.h)
+	var sumRowCount int64
```

Comment:
I do not think this approach will work.

Result returned here should contain row count just for single statement, not for them all.

Also you did not handle the Stmt.Query down below. User might be using Stmt.Query instead of Stmt.Exec, or both while fetching rowsets.

I also think (I did not actually test that), that SQLMoreResults advances to the next rowset. You cannot call SQLMoreResults multiple times here, otherwise you will miss them all.

Did you actually run all the tests to see if they work? I would be surprised if tests pass.

Reply:
Stmt.Query is not an issue since we will call closeByRows eventually which will close the cursor.

Yes all the tests that were passing before for me pass.

yes it does advance to the next rowset.

I assumed that Exec is only called when we are not expecting to see any results since it just returns row count.  But I am not the expert.